app/src/Lib/CLIHelper.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2022 - Daniel Weise <daniel.weise@concepts-and-training.de> - Extended GPL, see LICENSE */
+
+namespace CaT\Doil\Lib;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+trait CLIHelper
```

Comment:
This trait feels like a smell. Disparate functionality with a generic name. Please try to remove it, discuss with me or put it on the roadmap for later.

Reply:
removed the trait. Done